de.metas.acct.base/src/main/java-legacy/org/compiere/process/FactAcctReset.java

```diff
@@ -216,7 +216,7 @@ private void delete (String TableName, int AD_Table_ID)
 		else if (AD_Table_ID ==getTableId(I_M_InOut.class))
 			docBaseType = "IN ('" + X_C_DocType.DOCBASETYPE_MaterialDelivery
 				+ "','" + X_C_DocType.DOCBASETYPE_MaterialReceipt + "')";
-		else if (AD_Table_ID == MPayment.Table_ID)
+		else if (AD_Table_ID == getTableId(MPayment.class))
```

Comment:
please do
```suggestion
		else if (AD_Table_ID == getTableId(I_C_Payment.class))
```
instead. the less code places that refer to these legacy classes, the better